Makefile

```diff
@@ -37,6 +37,17 @@ cadctl-install-local-force:
 isclean: ## Validate the local checkout is clean. Use ALLOW_DIRTY_CHECKOUT=true to nullify
 	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." >&2 && exit 1)
 
+
+# will hold all of the checks that will run on the repo. this can be extracted to a script if need be
+.PHONY: checks
+checks: check-duplicate-error-messages
+
+# check-duplicate-error-messages will conform with 
+.PHONY: check-duplicate-error-messages
+check-duplicate-error-messages:
+	@(test $$(grep -Ir errors.New . | grep -v -e ./CONTRIBUTING.md | wc -l) -eq 0) || (echo "please use fmt.Errorf and not errors.New" >&2 && exit 1)
```

Comment:
Learning question: Why do we prefer fmt.Errorf over errors.New()?
Do we just want to get rid of injecting another dependency in our package that already uses fmt?

Reply:
not quite, it's just that fmt.Errorf allows for errors.New AND wrapping errors `fmt.Errorf("oh no! %w", err)`